cpp/scripts/run-clang-format.py

```diff
@@ -62,73 +36,85 @@ def parseArgs():
     argparser.add_argument("-regex", type=str,
                            default=r"[.](cu|cuh|h|hpp|cpp)$",
                            help="Regex string to filter in sources")
-    argparser.add_argument("-onlyChangedFiles", default=False,
-                           action="store_true",
-                           help="Run the formatter on changedFiles only.")
-    argparser.add_argument("-srcdir", type=str, default=".",
-                           help="Path to directory containing the dirs.")
+    argparser.add_argument("-ignore", type=str, default=r"cannylab/bh[.]cu$",
+                           help="Regex used to ignore files from matched list")
     argparser.add_argument("dirs", type=str, nargs="*",
                            help="List of dirs where to find sources")
     args = argparser.parse_args()
     args.regexCompiled = re.compile(args.regex)
+    args.ignoreCompiled = re.compile(args.ignore)
+    ret = subprocess.check_output("%s --version" % args.exe, shell=True)
+    ret = ret.decode("utf-8")
+    version = VersionRegex.match(ret)
+    if version is None:
+        raise Exception("Failed to figure out clang-format version!")
+    version = version.group(1)
+    if version != "8.0.0":
+        raise Exception("clang-format exe must be v8.0.0 found '%s'" % version)
     return args
 
 
-def isNewer(src, dst):
-    if not os.path.exists(dst):
-        return True
-    a = os.path.getmtime(src)
-    b = os.path.getmtime(dst)
-    return a >= b
+def list_all_src_files(fileRegex, ignoreRegex, srcdirs, dstdir, inplace):
```

Comment:
python vars should be underscored rather than camelCased, so `file_regex`, etc.

Reply:
Done